src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -220,9 +208,16 @@ public UInt160 GetCommitteeAddress(StoreView snapshot)
             return Contract.CreateMultiSigRedeemScript(committees.Length - (committees.Length - 1) / 2, committees).ToScriptHash();
         }
 
-        private IEnumerable<ECPoint> GetCommitteeMembers(StoreView snapshot, int count)
+        private IEnumerable<ECPoint> GetCommitteeMembers(StoreView snapshot)
         {
-            return GetCandidatesInternal(snapshot).OrderByDescending(p => p.Votes).ThenBy(p => p.PublicKey).Select(p => p.PublicKey).Take(count);
+            decimal votersCount = (decimal)new BigInteger(snapshot.Storages[CreateStorageKey(Prefix_VotersCount)].Value);
+            decimal VoterTurnout = votersCount / (decimal)TotalAmount;
+            if (VoterTurnout < EffectiveVoterTurnout)
+                return Blockchain.StandbyCommittee;
```

Comment:
In fact this is a good and smart protection, @erikzhang.

Reply:
Thus, we need at least 200000 NEO's voting

Reply:
Any big difference with `MinVotesOfCommittee = 10000`?

Reply:
What do you mean? @Tommo-L

Reply:
It is the same as `MinVotesOfCommittee = 20000000`. But the difference is that if we modify `TotalAmount` in a private chain, there is no need to modify `MinVotesOfCommittee`.